### DISCUSSION TITLE
With UGS no movement in XYZ

### DISCUSSION THREAD

**Discussion by Johan1974** [winder/Universal-G-Code-Sender#1950] [Help]
Hello,

I have installed a CNC 3018 Pro and it is working fne with Candle or Easel. I can move my XYZ around. 

With UGS i can make a connection with my machine but i am unable to move XYZ around. Whn i try to move it is only making a humming sound like it is stuck. 

When i turn back to Candle or Easel everything works fine again.  

I am using Windows 10,  any idea what this might be? 

Thanks!

Johan

**Comment by breiler** [comment]
Make sure your jogging feed speed is large enough. I think that it defaults to 10mm/minute which is a bit low.
